src/form/FormInput.js

```diff
@@ -127,7 +127,7 @@ styles = StyleSheet.create({
         height: 36,
       }
     }),
-    width: width,
+    width: width * 0.9,
```

Comment:
I believe this is addressing Issue #239. 
I would prefer to have this in a different PR. Also look at [PR 265](https://github.com/react-native-training/react-native-elements/pull/265/files) as this is an attempt to fix the same issue.